lib/spack/spack/bootstrap/_common.py

```diff
@@ -213,7 +213,8 @@ def _root_spec(spec_str: str) -> str:
     if str(spack.platforms.host()) == "darwin":
         spec_str += " %apple-clang"
     elif str(spack.platforms.host()) == "windows":
-        spec_str += " %msvc"
+        # TODO (johnwparent): Remove version constraint when clingo patch is up
+        spec_str += " %msvc@19.37"
```

Comment:
```suggestion
        spec_str += " %msvc@:19.37"
```
?

Reply:
Thank you, yes, that is what I had meant to express.